docs/src/main/asciidoc/vault.adoc

```diff
@@ -492,6 +492,45 @@ You should see:
 {password=connor}
 ----
 
+[[totp]]
+== TOTP Secrets Engine
+
+The TOTP secrets engine generates time-based credentials according to the TOTP standard.
```

Comment:
Hmm, I've lost a comment, proposal was to replace `The TOTP secrets engine` with a link, https://www.vaultproject.io/docs/secrets/totp/[VaultTOTP] and drop "You can read more about..."